### ISSUE TITLE
⌨ Mobile headings

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#14481]
### ⌨ Accessibility and Expected Behaviour

Issue reported: Headings in mobile view that span two lines are being treated as multiple headings. 

This adds additional navigational points when jumping from heading to heading. 

An example of this is "IT troubleshooting level intermediate assessed during initial application". "Required" appears on the next line and is treated as a separate heading.

### 🕵️ Details

Reported when applying for a job

Operating system: IOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

VoiceOver

### 🙋‍♀️ Proposed Solution

Headings should be one item, even when on separate lines.
